db/migrate/20210306210627_create_games.rb

```diff
@@ -0,0 +1,19 @@
+class CreateGames < ActiveRecord::Migration[6.1]
+  def change
+    create_table :games do |t|
+      t.text :against_team
+      t.timestamp :date
+      t.integer :goals_for
+      t.integer :goals_against
+      t.boolean :is_overtime
+      t.integer :powerplay_attemps
+      t.integer :killed_penalties
+      t.integer :total_penalties
+      t.text :city
+      t.text :ring_name
```

Comment:
I'm like 99% certain this is actually "rink" but you know what, good enough for now